tests/jballerina-integration-test/src/test/resources/configurables/configStructuredTypesProject/Config.toml

```diff
@@ -0,0 +1,210 @@
+
+[configStructuredTypes.employee]
+name="manu"
+id=101
+
+[configStructuredTypes.employee1]
+name="riyafa"
+id=999
+
+[[configStructuredTypes.employeeTable]]
+name="hinduja"
+id=102
+
+[[configStructuredTypes.employeeTable]]
+name="manu"
+id=100
+
+[[configStructuredTypes.employeeTable1]]
+name="waruna"
+id=2
+
+[[configStructuredTypes.employeeTable1]]
+name="manu"
+id=7
+
+[configStructuredTypes.person]
+name="gabilan"
+id=101
+
+[[configStructuredTypes.personTable]]
+name="hinduja"
+id=102
+
+[[configStructuredTypes.personTable]]
+name="manu"
+id=100
+
+[[configStructuredTypes.personTable1]]
+name="waruna"
+id=4
+
+[[configStructuredTypes.personTable1]]
+name="gabilan"
+id=5
+
+
+[configStructuredTypes.manager]
+name="hinduja"
+id=107
+
+[[configStructuredTypes.managerTable]]
+name="gabilan"
+id=101
+
+[[configStructuredTypes.managerTable]]
+name="riyafa"
+id=102
+
+[configStructuredTypes.teacher]
+name="gabilan"
+id=888
+
+[[configStructuredTypes.teacherTable]]
+name="manu"
+id=77
+
+[[configStructuredTypes.teacherTable]]
+name="riyafa"
+id=88
+
+
+[configStructuredTypes.farmer]
+name="waruna"
+id=999
+
+[[configStructuredTypes.farmerTable]]
+name="waruna"
+id=444
+
+[[configStructuredTypes.farmerTable]]
+name="hinduja"
+id=888
+
+
+[configStructuredTypes.mod2.doctor]
+name="waruna"
+
+
+[[configStructuredTypes.mod2.doctorTable]]
+name="hinduja"
+id=100
+
+[[configStructuredTypes.mod2.doctorTable]]
+name="riyafa"
+id=105
+
+
+[configStructuredTypes.mod2.student]
+name="riyafa"
+
+
+[[configStructuredTypes.mod2.studentTable]]
+name="manu"
+id=100
+
+[[configStructuredTypes.mod2.studentTable]]
+name="riyafa"
+id=105
+
+[configStructuredTypes.mod2.employee]
+name="manu"
+id=101
+
+[configStructuredTypes.mod2.employee1]
+name="waruna"
+id=404
+
+[[configStructuredTypes.mod2.employeeTable]]
+name="hinduja"
+id=102
+
+[[configStructuredTypes.mod2.employeeTable]]
+name="manu"
+id=100
+
+[[configStructuredTypes.mod2.employeeTable1]]
+name="gabilan"
+id=2
+
+[[configStructuredTypes.mod2.employeeTable1]]
+name="riyafa"
+id=3
+
+[configStructuredTypes.mod2.person]
+name="gabilan"
+id=101
+
+[[configStructuredTypes.mod2.personTable]]
+name="hinduja"
+id=102
+
+[[configStructuredTypes.mod2.personTable]]
+name="manu"
+id=100
+
+
+[[configStructuredTypes.mod2.personTable1]]
+name="hinduja"
+id=7
+
+[[configStructuredTypes.mod2.personTable1]]
+name="waruna"
+id=8
+
+[configStructuredTypes.mod2.manager]
+name="hinduja"
+id=107
+
+[[configStructuredTypes.mod2.managerTable]]
+name="gabilan"
+id=101
+
+[[configStructuredTypes.mod2.managerTable]]
+name="riyafa"
+id=102
+
+[configStructuredTypes.mod2.teacher]
+name="hinduja"
+id=11
+
+[[configStructuredTypes.mod2.teacherTable]]
+name="gabilan"
+id=66
+
+[[configStructuredTypes.mod2.teacherTable]]
+name="riyafa"
+id=77
+
+[configStructuredTypes.mod2.farmer]
+name="manu"
+id=22
+
+[[configStructuredTypes.mod2.farmerTable]]
+name="riyafa"
+id=555
+
+[[configStructuredTypes.mod2.farmerTable]]
+name="hinduja"
+id=666
```

Comment:
Shall we add new lines at the end of files?
https://github.com/ballerina-platform/ballerina-lang/pull/29865/files#diff-a009ac1d9eabca148c19c0eeb5cd989275fb18ea6195861ed0c7f4973fcf493a